### ISSUE TITLE
[BUG] Small changes in "current_consumption" generate a DB update

### ISSUE THREAD

**Issue by chemelli74** [StyraHem/ShellyForHASS#316]
## Environment

- ShellyForHASS release with the issue: 0.1.9.b1
- Last working ShellyForHASS release (if known): -
- Home Assistant Core release with the issue:  0.110.4
- Operating environment (Home Assistant/Supervised/Docker/venv): Docker

## Describe the bug

I have a Shelly 1PM that flips from 0w to 1w continuously.
After enabling 2 digit decimals I noticed that it flips between 0.71 and 0.78.

## Steps to Reproduce

Hard to reproduce if you don't have a flipping device

## Expected behavior

As device flips between values that are all greater than 0.51 I would expect device to report always 1w and not flip, thus limiting DB updates.

Simone

**Comment by chemelli74** [comment]
Added a couple of debug lines and found what follows:

```
2020-06-26 17:20:11 WARNING (CoAP) [pyShelly] consumption = 0.92, rounded = 1
2020-06-26 17:20:11 WARNING (SyncWorker_0) [custom_components.shelly.sensor] state = 0.92
```

Seems that the rounded value is not passed by the library to the sensor code in ShellyForHASS.

Simone

**Comment by chemelli74** [comment]
Issue seems fixed in latest 0.1.9.
Closing for now.

Simone
